client/app/types/course/users.ts

```diff
@@ -0,0 +1,48 @@
+import { Permissions } from 'types';
```

Comment:
there is course_users.ts

Could we move everything here to that file and call it CourseUserXXX instead? The reason for this is because we need to differentiate courseUser and User in the future.